c2rust-analyze/src/rewrite/ty.rs

```diff
@@ -380,25 +423,26 @@ pub fn gen_ty_rewrites<'tcx>(
 
     assert_eq!(lty_sig.inputs.len(), hir_sig.decl.inputs.len());
     for (&lty, hir_ty) in lty_sig.inputs.iter().zip(hir_sig.decl.inputs.iter()) {
-        let rw_lty = relabel_rewrites(asn, rw_lcx, lty);
-        v.handle_ty(rw_lty, hir_ty);
+        let rw_lty = relabel_rewrites(&asn.perms(), &asn.flags(), rw_lcx, lty);
+        v.handle_ty(rw_lty, hir_ty, LifetimeType::Elided);
     }
 
     if let hir::FnRetTy::Return(hir_ty) = hir_sig.decl.output {
-        let rw_lty = relabel_rewrites(asn, rw_lcx, lty_sig.output);
-        v.handle_ty(rw_lty, hir_ty);
+        let rw_lty = relabel_rewrites(&asn.perms(), &asn.flags(), rw_lcx, lty_sig.output);
+        v.handle_ty(rw_lty, hir_ty, LifetimeType::Elided);
     }
 
     let hir_body_id = acx.tcx().hir().body_owned_by(ldid);
     let body = acx.tcx().hir().body(hir_body_id);
     intravisit::Visitor::visit_body(&mut v, body);
 
-    // TODO: wrap locals in `Cell` if `addr_of_local` indicates that it's needed
```

Comment:
Is this TODO outdated (and the Cell PR should have removed this comment), or am I missing wheres this PR is doing something to locals?

Reply:
the Cell PR should have removed this comment